eobcanka_deb2rpm.sh

```diff
@@ -38,6 +38,10 @@ rm "opt/eObcanka/lib/openssl1.1/libcrypto.so.1.1"
 sed -i '/^%postun/ i ln -fs /usr/lib64/libcrypto.so.1.1 /opt/eObcanka/lib/openssl1.1/libcrypto.so.1.1' "$SPEC"
 sed -i 's/^\(".*\/libcrypto.so.1.1"\)$/%ghost \1/' "$SPEC"
 
+# Require Czech locale package and remove Debian-specific local-gen
+sed -i '/^Group:/ a Requires: glibc-langpack-cs' "$SPEC"
+sed -i '/^##CZ language support/,+6d' "$SPEC"
```

Comment:
The piece of code we want to remove from the spec file is:
```
##CZ language support
ISLANGCSACTIVE=$(/usr/bin/locale -a|grep cs_CZ)

if [ -z $ISLANGCSACTIVE ] ; then
    locale-gen cs_CZ.utf8
fi
```

```suggestion
sed -i 'N;/^##CZ language support/,/locale-gen cs_CZ.utf8\nfi/d' "$SPEC"
```
^^ This sed command removes the part that starts with `##CZ language support` and ends with the multi-line pattern `locale-gen cs_CZ.utf8\nfi`. I think this way it's a bit safer.
```